src/AppHead.tsx

```diff
@@ -0,0 +1,45 @@
+import Head from 'next/head';
+
+interface AppHeadProps {
+  title: string;
+  importElk: boolean;
+  description: string;
+  importPrettier: boolean;
+  ogImageUrl: string | undefined;
+}
+
+export const AppHead = (props: AppHeadProps) => {
+  return (
+    <Head>
+      <link rel="apple-touch-icon" href="/viz/favicon@256.png" />
+      <link rel="icon" href="/viz/favicon.png" />
+      <title>{props.title}</title>
+      <meta name="description" content={props.description} />
+      {props.importPrettier && (
+        <>
+          <script src="https://unpkg.com/prettier@2.3.2/standalone.js"></script>
+          <script src="https://unpkg.com/prettier@2.3.2/parser-typescript.js"></script>
+        </>
+      )}
+      {props.importElk && (
+        <script src="https://unpkg.com/elkjs@0.7.1/lib/elk.bundled.js"></script>
+      )}
+
+      <meta property="og:type" content="website" />
+      <meta property="og:url" content={`https://stately.ai/viz`} />
+      <meta property="og:title" content={props.title} />
+      <meta property="og:description" content={props.description} />
+      {props.ogImageUrl && (
+        <meta property="og:image" content={props.ogImageUrl} />
+      )}
+
+      <meta property="twitter:card" content="summary_large_image" />
+      <meta property="twitter:url" content={`https://stately.ai/viz`} />
+      <meta property="twitter:title" content={props.title} />
+      <meta property="twitter:description" content={props.description} />
+      {props.ogImageUrl && (
+        <meta property="twitter:image" content={props.ogImageUrl} />
+      )}
```

Comment:
Those are somewhat redundant because Twitter already understands og properties. The only thing that we need is `twitter:card`, as that is not part of the OpenGraph standard. 

There is no harm in including both lists but if we can save some bytes - I would be in favor of that :P

Reply:
Does it? I was under the impression that Twitter needs a different og set, based on my limited research. But if it can be deleted, we should!

Reply:
My research was also limited 😅  but based on a single source on the internet and vague memory of it being true - it should be supported. We can always add it back if it turns out not be the case

Reply:
Done!